src/applications/vaos/tests/new-appointment/components/VAFacilityPage/VAFacilityPageV2.multi.unit.spec.jsx

```diff
@@ -1746,4 +1763,45 @@ describe('VAOS <VAFacilityPageV2> multiple facilities variant test', () => {
     const firstRadio = screen.container.querySelector('.form-radio-buttons');
     expect(firstRadio).to.contain.text('ABC facility');
   });
+
+  it('should fire variant shown and default sort method events when variant shown', async () => {
```

Comment:
I couldn't find a good way to test this. For some reason moving between pages doesn't work with our helper, even when you render the NewAppointment component. So I've mocked out the state to get the events to fire.